server/routes/requests.js

```diff
@@ -322,4 +323,53 @@ router.get('/getConfirmed/:id/:timeOffset', passport.authenticate('jwt', {sessio
     }
 })
 
+router.patch('/actionItems/:id/:timeOffset', passport.authenticate('jwt', {session: false}), async (req, res, next) => {
+    let alumniId = req.params.id;
+    let timeOffset = req.params.timeOffset;
+    let requestId = req.body.requestId;
+    let actionItems = req.body.actionItems;
+    let conditions = ['Awaiting Confirmation', 'Confirmed', 'Completed']
+    let requests = []
+    try {
+        let request = await requestSchema.findById(requestId);
+        request.actionItems = actionItems;
```

Comment:
This might need to be `request.actionsItems.push(actionItems)` since an actionItem is an object, but the field on the schema is an array